### PR TITLE
audit round 2 suggestions

### PR DESCRIPTION
**Pull request by mrnerdhair**

### CODE REVIEW

File: src/libraries/Ownable.sol [hunk 1]
```diff
@@ -41,8 +49,10 @@ contract Ownable is IOwnable {
         @notice renounce ownership to 0 address
      */
     function renounceOwner() public virtual override onlyOwner {
+        newOwner = address(0);
```
Comment (to-sh-1) [inline src/libraries/Ownable.sol]: Would you mind removing this function as we discussed?

**Review by to-sh-1 — COMMENTED** [review]
Looks good to me.  Suggested to remove the renounceOwner function.
Gonna hold off on approval, because I would like to have this in a separate PR so Zokyo can have two different PR's to see the changes they requested and our changes

File: src/libraries/Ownable.sol (hunk 1 above)
Comment (mrnerdhair) [reply src/libraries/Ownable.sol]: right, check, 1sec

Comment (mrnerdhair) [reply src/libraries/Ownable.sol]: done

**Comment by mrnerdhair** [comment]
Removed renounceOwner, and changed the base branch to develop so the PRs can go in separately.

**Pull request merged**